core/entity/bill.py

```diff
@@ -24,5 +24,17 @@ class Bill(BaseModel):
     created_user_id: Optional[int] = None
     updated_user_id: Optional[int] = None
     created_user: Optional[Username] = None
+    class Config():
+        orm_mode = True
+        
+class BillSmall(BaseModel):
```

Comment:
id, patient, patient_name, patient_phone, patient_address, total_amount, payment and created_time is required.